vast/integration/vast_integration_suite.yaml

```diff
@@ -283,14 +283,34 @@ tests:
       - command: -N export ascii
         input: queries/multi_addr.txt
 
+  Pivot with PCAP:
+    tags: [node, pivot, pcap]
+    steps:
+      - command: -N import suricata
+        input: data/pcap/suricata/eve.json.gz
+      - command: -N import zeek
+        input: data/pcap/zeek/conn.log.gz
+      - command: -N import zeek
+        input: data/pcap/zeek/files.log.gz
+      - command: -N import zeek
+        input: data/pcap/zeek/http.log.gz
+      - command: -N import zeek
+        input: data/pcap/zeek/packet_filter.log.gz
+      - command: -N import zeek
+        input: data/pcap/zeek/ssh.log.gz
+      - command: -N pivot --format=zeek zeek.conn '#type ~ /suricata.*/ && dest_ip == 72.247.178.18'
+        transformation: awk '!/^#(open|close)/'
+      - command: -N pivot zeek.conn '#type ~ /zeek.http/ && method == "GET"'
+        transformation: awk '!/^#(open|close)/'
+
   Taxonomy queries:
     tags: [concepts, models]
     fixture: ServerTester
     steps:
       - command: import -b zeek
-        input: data/zeek/pcap-conn.log.gz
+        input: data/pcap/zeek/conn.log.gz
```

Comment:
I think we should try improve the structure of the `data` directory. I propose to group all files by format and add a `from-pcap` subdir where appropriate. Then this file would be at `data/zeek/from-pcap/example.conn.log.gz`.

Reply:
Should we maybe do this in a separate PR?

Reply:
I've written a story.